CONTRIBUTING.md

```diff
@@ -17,3 +17,32 @@ license.
 By contributing documentation, examples, or any other non-code assets you agree
 to license your contribution under the CC BY 3.0 license. Attribution shall be
 given according to the current bylaws of this group.
+
+# Merge & Access Policy
+
+All Editors, Coordinators and Sponsors of specifications in draft stage have
+access to push to this repository. Secretaries have full administrative and
+push access. The master branch is protected and therefore cannot be forced
+pushed to.
+
+Guidelines for merging
+* Never force push
```

Comment:
If it's about master branch, it is not possible technically because branch protection.